seekinternational/notification-library
# Feature/circleci integration

What does this PR do?
---------------------
Add circleci integration to enable status check for:

1) Linting
2) Testing
3) Build check

How can I test this?
--------------------
1) Deploy and check CircleCI status
